### PR TITLE
More links

### PR DESCRIPTION
**Pull request by gislenius**
@dknochen #774 is a prerequisite that should first be reviewed and merged (will reduce the diff).

### CODE REVIEW

**Comment by gislenius** [comment]
#774 is a prerequisite

**Comment by gislenius** [comment]
@dknochen back to you!

File: queries/public/api/components.jq [hunk 1]
```diff
@@ -140,7 +106,8 @@ let $res as object* :=
                AcceptanceDatetime : sec-filings:acceptance-dateTimes($archive),
                FormType : $archive.Profiles.SEC.FormType,
                Components : [
-                    for $component in sec-networks:summaries($r)
+                    for $r in $r
```
Comment (dknochen) [inline queries/public/api/components.jq]: Could we rename this variable into 2 distinguishable names? E.g. `$components-group` and `$component`

File: queries/public/test/secxbrl/components.jq [hunk 2]
```diff
@@ -111,17 +111,30 @@ local:check({
...
+        "SpreadSheet" : "http://rendering.secxbrl.info/#?url=http%3A%2F%2F"||request:server-name()||"%3A"||request:server-port()||"%2Fv1%2F_queries%2Fpublic%2Fapi%2Fspreadsheet-for-component.jq%3Ftoken%3D"||$config:test-token||"%26aid%3D0000021344-13-000039%26format%3D%26role%3Dhttp%253A%252F%252Fwww.thecoca-colacompany.com%252Frole%252FDocumentAndEntityInformation%26profile-name%3Dsec",
+      "ReportElements" : "http://"||request:server-name()||":"||request:server-port()||"/v1/_queries/public/api/report-elements.jq?token=foobar&aid=0000021344-13-000039&format=&role=http%3A%2F%2Fwww.thecoca-colacompany.com%2Frole%2FDocumentAndEntityInformation&profile-name=sec",
+      "ModelStructure" : "http://"||request:server-name()||":"||request:server-port()||"/v1/_queries/public/api/modelstructure-for-component.jq?token="||$config:test-token||"&aid=0000021344-13-000039&format=json&role=http%3A%2F%2Fwww.thecoca-colacompany.com%2Frole%2FDocumentAndEntityInformation&profile-name=sec"
```
Comment (dknochen) [inline queries/public/test/secxbrl/components.jq]: could we spread these over multiple lines? max line length should be 100 characters.

**Comment by dknochen** [comment]
like this improvement :+1:

**Comment by dknochen** [comment]
very cool :)

**Pull request merged**